client/helpers/index.js

```diff
@@ -75,3 +76,12 @@ export const flattenReveals = (fields, values) => {
     ])
   }, []);
 };
+
+export const changed = (fields, latest, granted) => {
+  if(fields.some(k=> latest.some(l=> l.includes(k)))) {
```

Comment:
space after `if`, space after `k`, space after `l` - lint should be picking this up 🤔